mlflow/R/mlflow/tests/testthat/test-model-registry.R

```diff
@@ -75,6 +75,68 @@ test_that("mlflow can register a model with tags and description", {
   )
 })
 
+test_that("mlflow can get a registered model", {
+  with_mock(.env = "mlflow",
+    mlflow_rest = function(...) {
+      args <- list(...)
+      expect_true(paste(args[1:2], collapse = "/") == "registered-models/get")
+      expect_equal(args$verb, "GET")
+      expect_equal(args$query$name, "test_model")
+      return(list(
+        registered_model = list(
+          name = "test_model"
+        )
```

Comment:
I think we can fit some of this on one line.

Reply:
Done